### PR TITLE
Feat(eos_cli_config_gen): Add support for deadtime configuration to radius_server

### LINKED ISSUE #4268: Add deadtime configuration to radius_server
### Enhancement summary

We would like to also be able to configure the deadtime parameter for our RADIUS servers with AVD. Currently only timeout and retransmit can be adjusted through AVD. I propose adding another variable to the radius_server model to also be able to adjust the deadtime.

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Configuring the deadtime for a RADIUS server, so that unresponsive servers do not lead to repeated attempts to contact them. 

### Describe the solution you would like

I think the best implementation would be to add to the radius_server model, as another possible value for the hosts.

```
radius_server:
  hosts:
    - host: <str; required; unique>
      deadtime: <int, 0 - 1000>
```

### Describe alternatives you have considered

Right now, we are getting around this by using eos_cli to write this part of the configuration directly.

```
eos_cli:
    radius-server deadtime 5
```
However, this makes it harder to see the entire radius_server variables at a glance.

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Shivani-gslab**
## Change Summary

Add support for deadtime configuration to radius_server

## Related Issue(s)

Fixes #4268 

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
```
radius_server:
      # Time in minutes to skip a non-responsive server.
      deadtime: <int; 1-1000>
```

## How to test
Molecule test or check the EOS CLI.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by MaheshGSLAB — APPROVED** [review]

**Review by laxmikantchintakindi — APPROVED** [review]

**Review by ClausHolbechArista — DISMISSED** [review]
Blocking merge until 5.0.0 has been released.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/radius_server.schema.yml
Comment (gmuloc) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/radius_server.schema.yml]: move it before dynamic_authorization in the schema

**Review by gmuloc — CHANGES_REQUESTED** [review]

**Review by gmuloc — APPROVED** [review]
LGTM - can merge once CI completes

**Pull request merged**
